app/components/Dashboard/BestNewMusicTab/BestNewMusicMenu/BestNewMusicCard/index.js

```diff
@@ -1,40 +1,110 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Icon } from 'semantic-ui-react';
 
+import { removeQuotes, createLastFMLink } from '../../../../../utils';
+import { favoriteTrackShape } from '../../../../../constants/propTypes';
 import styles from './styles.scss';
 
+function toFavoriteTrack({ artist, title, thumbnail }) {
+  const name = removeQuotes(title);
+  const url = createLastFMLink(artist, name);
+
+  return {
+    name,
+    artist: {
+      name: artist
+    },
+    url,
+    image: ['small', 'medium'].map(size =>
+      ({
+        '#text': thumbnail,
+        size
+      }))
+  };
+}
+
+const FavoriteIcon = ({ isFavorite, onClick }) =>
+  <Icon className={styles.card_favorite} name={isFavorite ? 'heart' : 'heart outline'} size='large' onClick={onClick} />;
```

Comment:
I'm having issues with `star outline` `Icon`. For me, no version of `semantic-ui-react` provides the font with an outline version of `star`. I'd be grateful if someone could check if it's only my problem. Used `heart` icon temporarily.

Reply:
Resolved by using `disabled` prop of `Icon` component which renders favorite/non-favorite as follows:

![image](https://user-images.githubusercontent.com/8076673/63199929-abf74780-c07f-11e9-81fb-5ed863287986.png)